agave/filters.py

```diff
@@ -20,3 +23,68 @@ def generic_query(query: QueryParams) -> Q:
     if 'count' in fields:
         del fields['count']
     return filters & Q(**fields)
+
+
+def get_by_id(cls, id: str):
```

Comment:
why would `get_by_id` and `get_by_id_and_user` be different? Couldn't you have `user_id` as an optional parameter?

Reply:
la función ahora es get()